src/sys/socket/sockopt.rs

```diff
@@ -238,6 +238,17 @@ cfg_if! {
 }
 sockopt_impl!(Both, IpMulticastTtl, libc::IPPROTO_IP, libc::IP_MULTICAST_TTL, u8);
 sockopt_impl!(Both, IpMulticastLoop, libc::IPPROTO_IP, libc::IP_MULTICAST_LOOP, bool);
+cfg_if! {
```

Comment:
You shouldn't use `cfg_if` if you're only going to have one clause.  Instead, just do
```rust
#[cfg(target_os = "linux")]
whatever
```

Reply:
Did you forget to push this change?